packages/fx-core/tests/plugins/resource/function/unit/expectedBicepFiles/function_test.bicep

```diff
@@ -0,0 +1,104 @@
+param functionServerfarmsName string
+param functionAppName string
+@minLength(3)
+@maxLength(24)
+@description('Name of Storage Accounts for function backend.')
+param functionStorageName string
+param aadClientId string
+@secure()
+param aadClientSecret string
+param m365TenantId string
+param applicationIdUri string
+
+param frontendHostingStorageEndpoint string
+
+var oauthAuthorityHost = environment().authentication.loginEndpoint
+var teamsAadIds = '1fec8e78-bce4-4aaf-ab1b-5451cc387264;5e3ce6c0-2b1f-4285-8d4b-75ee78787346'
```

Comment:
Please refer to *packages\fx-core\templates\plugins\resource\simpleauth\bicep\simpleAuth.template.bicep* and use readable variables to refer to this two client id.